notebooks/idr0094-r.ipynb

```diff
@@ -0,0 +1,1441 @@
+       "20  Monensin sodium                  0.61355452\n",
+       "21  Tioguanine                       0.99448693\n",
+       "22  Abamectin (avermectin B1a)       0.32622173\n",
+       "23  Penfluridol                      0.31639973\n",
+       "24  Cyclosporine                    15.81528194\n",
+       "25  Fosaprepitant dimeglumine salt   7.99370322\n",
+       "26  PF-04929113 (SNX-5422)           0.07932985\n",
+       "27  GSK2636771                       0.08913933\n",
+       "28  Thonzonium bromide               1.28177015\n",
+       "29  Proflavine hemisulfate           2.10174716\n",
+       "30  Rociletinib (CO-1686, AVL-301)   0.07100162\n",
+       "⋮   ⋮                              ⋮           \n",
+       "303 SMI-4a                          0.83172680 \n",
+       "304 Endoxifen HCl                  11.98357416 \n",
+       "305 RG7112 (RO5045337)              0.05247987 \n",
+       "306 AZD3514                         1.32979979 \n",
+       "307 OSI-930                         1.07842382 \n",
+       "308 4-CMTB                          0.29877329 \n",
+       "309 PIK-75                          0.12723582 \n",
+       "310 QNZ (EVP4593)                   0.34679274 \n",
+       "311 AZD8055                         9.93249504 \n",
+       "312 WYE-354                         0.22095276 \n",
+       "313 Tocofersolan                    0.00344641 \n",
+       "314 Ku-0063794                      4.18122741 \n",
+       "315 GDC-0349                        0.04342338 \n",
+       "316 JNJ 1661010                     9.86323430 \n",
+       "317 CCT128930                       0.05393318 \n",
+       "318 AZD3463                         0.17872181 \n",
+       "319 OSI-027                         0.04038401 \n",
+       "320 Wnt-C59 (C59)                   3.12562789 \n",
+       "321 Torin 1                         0.18016227 \n",
+       "322 DBeQ                            0.01993226 \n",
+       "323 AZD7545                         0.04587587 \n",
+       "324 WAY-600                         1.36029155 \n",
+       "325 GSK2656157                      1.43811294 \n",
+       "326 Skepinone-L                    12.21639766 \n",
+       "327 MK-5108 (VX-689)                0.51440429 \n",
+       "328 GSK1059615                      0.44212409 \n",
+       "329 BMS 794833                      0.19518495 \n",
+       "330 SNX-2112                        0.28457287 \n",
+       "331 SC144                           0.11833250 \n",
+       "332 R547                            2.38173685 "
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ic <- vector()\n",
+    "cn <- vector()\n",
+    "for(i in 1:nrow(compounds)) {\n",
+    "    c <- compounds[i,]\n",
+    "    if (c != \"\") {\n",
+    "      data <- filter_data(df, c)\n",
+    "      results <- calculate_IC50(data)\n",
+    "      ic <- c(ic, results$ic50)\n",
+    "      cn <- c(cn, c) \n",
```

Comment:
the code is fine as is I suppose, but for the printout of the table in the notebook, a bit more verbose variant of the column headers would be appropriate imho

![Screenshot 2021-04-14 at 15 06 44](https://user-images.githubusercontent.com/2478303/114724206-3bf90300-9d33-11eb-8708-a7aac169a16c.png)